docs/infrastructure_and_maintenance/security/development_security.md

```diff
@@ -145,3 +145,18 @@
                 entry_point: lexik_jwt_authentication.jwt_token_authenticator
             stateless: true
 ```
+
+In order for the application to be able to generate JWT tokens, a [PEM encoded keyset needs to be generated](https://symfony.com/bundles/LexikJWTAuthenticationBundle/current/index.html#generate-the-ssl-keys) using the command:
+
+```bash
+php bin/console lexik:jwt:generate-keypair
+```
+
+The generated key pair will be stored in `config/jwt`
+
+!!! note "[[= product_name_cloud =]]"
+
```

Comment:
To generate and store tokens on [[= product_name_cloud =]], you must define `config/jwt` as a volume in `.platform.app.yaml`. In a cluster setup, ensure the key pair is the same on all servers. You can achieve this by using a network share or by manually copying the key pair to all servers.